src/parser/set_process_list.c

```diff
@@ -1,5 +1,37 @@
 #include "../../include/minishell.h"
 
+static void word_token_to_char_list(t_process *ps_list)
+{
+	int		count;
+	char	**new_argv;
+	t_token	*token_curr;
+	int		idx;
+
+	idx = 0;
+	count = word_type_count(ps_list->cmd_line);
+	new_argv = (char **)malloc(sizeof(char *) * (count + 1));
```

Comment:
```suggestion
	new_argv = safe_malloc(sizeof(char *) * (count + 1));
```

we didn't write type in front of malloc, and use safe_malloc

Reply:
great